lmp/bb-config.sh

```diff
@@ -49,6 +51,15 @@ EULA_stm32mp15disco="1"
 EULA_stm32mp1eval="1"
 EULA_stm32mp15eval="1"
 
+# meta-lmp requires this value to be a script and not a script with arguments.
+# This creates a wrapper that handles the arguments
+fetch_creds=$(mktemp --suffix=.sh)
```

Comment:
Maybe if we set `H_RUN_URL=${H_RUN_URL}` in `conf/local.conf` then we will not need this dynamic scripting? Similar case (`GARAGE_TARGET_VERSION = "${H_BUILD}"` at #94.

Reply:
you are probably right. @ricardosalveti - is mike's suggestion preferable for this?

Reply:
I think it's not so important to be a blocker for //M.

Reply:
IMO we should avoid using `mktemp` in bitbake variable because it breaks the sstate cache, they generate a different values in each run. Something like `${HERE}/fetch-root-meta-helper.sh` look better to me.

This variable seems to only affect the tasks of the final target image that always runs on the CI build (do not use sstate-cache) but avoid the `mktemp` usage on variable names is good police in my opinion.

Reply:
The script to looks good to me, it's just the name that worried me because they are not reproducible

Reply:
rebased on latest master and then fixed in force-push